docs/reference/query-dsl/geo-shape-query.asciidoc

```diff
@@ -111,9 +111,17 @@ The following is a complete list of spatial relation operators available:
 
 * `INTERSECTS` - (default) Return all documents whose `geo_shape` field
 intersects the query geometry.
+An example of when to use this would be checking if a road (mapped as a
+`linestring`) overlaps the boundaries of a city (mapped as a `polygon`),
```

Comment:
Intersects doesn't just have to overlap the boundaries of two shapes; it can be

- the boundary of one shape overlapping the boundary of another
- the interior of one shape overlapping the boundary of another
- the interior of one shape overlapping the interior of another (i.e. `WITHIN`)

[These illustrations from ESRI do a reasonable job of illustrating](http://edndoc.esri.com/arcsde/9.0/general_topics/understand_spatial_relations.htm).

Maybe change this to 

> An example of when to use this would be checking if a road (mapped as a `polygon`) overlaps with a city (mapped as a `polygon`); The two shapes would intersect if:
> 1. the boundary of the road overlaps with the boundary of the city
> 2. the interior of the road overlaps with the boundary of the city
> 3. the boundary of the road overlaps with the interior of the city
> 4. the interior of the road overlaps with the interior of the city

Maybe some diagrams would help? Happy to draw some up as SVGs